libs/designsystem/card/src/card.component.scss

```
@@ -57,6 +57,24 @@
     box-shadow: none;
   }
 
+  &.outlined {
+    background-color: transparent;
+    outline: 1px solid utils.get-color('medium');
+    box-shadow: none;
+
+    &:has(kirby-card-header.flagged) {
+      outline: none;
+
+      .content-wrapper {
+        border-block-end: 1px solid utils.get-color('medium');
+        border-inline-start: 1px solid utils.get-color('medium');
+        border-inline-end: 1px solid utils.get-color('medium');
+        border-bottom-left-radius: 16px;
+        border-bottom-right-radius: 16px;
```

Should use same value/variable as the overall card border radius:
```suggestion
        border-bottom-left-radius: utils.$border-radius;
        border-bottom-right-radius: utils.$border-radius;
```